README.md

```diff
@@ -1,12 +1,39 @@
+
 [![Join the chat at https://gitter.im/fetch-amul-topicals/Lobby](https://badges.gitter.im/fetch-amul-topicals/Lobby.svg)](https://gitter.im/fetch-amul-topicals/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system
+## Table of Contents
+* Introduction
+    * Requirements
+* Installation Guide
+* Screenshots
+* License
+
+## Introduction
+A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system.
+
+### Requirements
+* Programming Language : Python (2 or 3)
+* Libraries : BeautifulSoup/ json/ Python Requests
```

Comment:
Minor edit needed:
- Check again if json and requests are already there or you need to install it.
- Slashes don't provide the correct meaning here, separate it by commas.

I'll merge right after this